pixeltable/tests/test_table.py

```diff
@@ -111,6 +114,24 @@ def test_create_image_table(self, test_client: pt.Client) -> None:
         for tup in pdf.itertuples():
             assert tup.img == tup.img_literal
 
+        # Test adding stored image transformation
+        tbl.add_column(rotated=tbl.img.rotate(30), stored=True)
+        assert(MediaStore.count(tbl.id) == 2 * n_sample_rows)
+
+        # Test MediaStore.stats()
+        stats = list(filter(lambda x: x[0] == tbl.id, MediaStore.stats()))
+        assert len(stats) == 2                 # Two columns
+        assert stats[0][2] == n_sample_rows    # Each column has n_sample_rows associated images
+        assert stats[1][2] == n_sample_rows
+
+        # Test version-specific MediaStore.delete
+        MediaStore.delete(tbl.id, version=1)
```

Comment:
You should do this with a tbl.revert() (which will remove the column and delete the media files). MediaStore.delete() shouldn't be called directly.